src/components/Tool/Import/Csv/uploadCsvFile.ts

```diff
@@ -182,7 +182,7 @@ export const useSupportedHeaders = () => {
 };
 
 export const useRequiredHeaders = () => {
-  const hardCodedRequiredHeaders = ['first_name', 'last_name', 'full_name'];
+  const hardCodedRequiredHeaders = ['first_name', 'last_name'];
 
   const [requiredHeaders, setRequiredHeaders] = useState(
```

Comment:
I don't fully understand the `useEffect` below, but right now it will run on every render. To only make it run once, it needs an empty dependency array `[]`.